apps/frontend/src/app/layout.tsx

```diff
@@ -17,7 +22,12 @@ export default function RootLayout({
 }>) {
   return (
     <html lang='hu'>
-      <body className={inter.className}>{children}</body>
+      <body className={`${inter.className} h-lvh flex flex-col justify-between`}>
```

Comment:
Does `h-lvh` look correct on mobile when you scroll down a lot and then back up a little?

Reply:
hmmm, i've changed it already to min-h-shv, although I am unable to test in the browser mobile tool

Reply:
I think you probably want dvh, at least thats what I used in paybasz

Reply:
With dvh, it adapts to the navbar disappearing

Reply:
I think you are unable to test this behavior in a desktop browser right now

Reply:
Nice, thanks, i have changed it to 'dvh'